Cargo.lock

```diff
@@ -4,8 +4,12 @@ version = "0.1.0"
 dependencies = [
  "bson 0.1.2 (git+https://github.com/kyeah/bson-rs)",
  "byteorder 0.3.10 (registry+https://github.com/rust-lang/crates.io-index)",
+ "libc 0.1.8 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
This is a generally good idea, but in a couple edges cases we'd have to not to that in order for things to work. The main one I'm thinking of is when a module with macros is imported in lib.rs/mod.rs; the macros will only be accessible for the modules declared *after* the import/declaration of the module with macros, so those ones would have to appear first.